api/src/controllers/reviews.ts

```diff
@@ -179,7 +189,7 @@ router.delete('/', async (req, res, next) => {
     res.json(status);
   }
   else {
-    res.json({ error: 'Must be an admin to delete reviews!' });
+    res.json({ error: 'Must be an admin or authorized user to delete reviews!' });
```

Comment:
Just a thought -- would it make more sense to say `review author` instead of `authorized user`?

I'll leave this up to you though, it's a minor nit 😅

Reply:
Done